### ISSUE TITLE
None of Databases is showing in "Connect to a Database" page

### ISSUE THREAD

**Issue by Hari-stackroute** [Kanaries/Rath#421]
I installed and ran the recent version of Rath-service, and tried to connect with postgres DB with existing tables to see how it reflects for my use-case.
None of the options are coming up, please help in this. same is happening with environment https://rath.kanaries.net/
Is the Database connectivity removed?

please see the screen shot
<img width="1140" alt="Screenshot 2024-09-12 at 5 57 26 PM" src="https://github.com/user-attachments/assets/d1e2a9e5-61e8-4117-b812-51061635812f">
<img width="1140" alt="Screenshot 2024-09-12 at 5 57 26 PM" src="https://github.com/user-attachments/assets/d1e2a9e5-61e8-4117-b812-51061635812f">

**Comment by ObservedObserver** [comment]
Yes, RATH had strated to remove all database connections on playground. but you can still setup the connection services locally with https://github.com/Kanaries/Rath/tree/master/services/connector.

If you are integrate RATH with your data infra, it is more recommended to manage the connection at your own infra platform, and the set loaded data to RATH with postMessage right now.
